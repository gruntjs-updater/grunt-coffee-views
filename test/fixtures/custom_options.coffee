{Html} = require 'coffee-views'

class CustomOptions extends Html
  mung: (params) ->
    @html ->
      @body ->
        @p params.title

module.exports = CustomOptions

