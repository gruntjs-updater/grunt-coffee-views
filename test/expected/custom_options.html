<html><body><p>Custom options</p></body></html>
